add narratives in usage (#56)
UBC-MDS/pygtracker: README.md

tracker.register_courses(courses)
tracker.record_grades(grades)
```

> 2. By specifying a list of courses, function `generate_course_statistics()` would allow you create a data frame with summary statistics, including `mean`, `1st-quantile`, `median` and `3rd-quantile` of grades so far.

Reviewer: ```suggestion
2. By specifying a list of courses, function `generate_course_statistics()` would allow you to create a data frame with summary statistics, including `mean`, `1st-quantile`, `median` and `3rd-quantile` of grades so far.
```

|---:|------------:|-------:|---------------:|---------:|---------------:|
|  0 |         511 |  87.87 |          86.91 |       88 |          88.96 |

> 3. Function `rank_courses()` generates a data frame of course ranking based on specified `method` (default is `mean`) in descending order by default. This would allow the user find which course students perform better and which they may struggle with so far.

Reviewer: ```suggestion
3. Function `rank_courses()` generates a data frame of course ranking based on specified `method` (default is `mean`) in descending order by default. This would allow the user to find which course students perform better and which they may struggle with so far.
```

|  0 |         522 |   91.29 |
|  1 |         511 |   87.87 |

> 4. To look into more details on performance of individual students, function `rank_students()` would generate a data fram with student ranking by grades based on specified courses (default is all courses completed so far) in descending order. Thus, the students who may need more help or guidance could be found.

Reviewer: ```suggestion
4. To look into more details on performance of individual students, function `rank_students()` would generate a data frame with student ranking by grades based on specified courses (default is all courses completed so far) in descending order. Thus, the students who may need more help or guidance could be found.
```

|  1 | tom          |   90.09 |      2.0 |
|  2 | mike         |   88.29 |      3.0 |

> 5. Lastly, based on function `suggest_grade_adjustment()`, the user is able to find adjusting suggestions by setting `benchmark` of courses, labs, or quizzes of a course, such that the final grades meet or exceed these bencmarks.

Reviewer: ```suggestion
5. Lastly, based on function `suggest_grade_adjustment()`, the user is able to find adjusting suggestions by setting `benchmark` of the whole course, labs, or quizzes of that course, such that the final grades meet or exceed these benchmarks.
```
